test/unit/models/classes/resources/ResourceWatcherTest.php

```
@@ -82,6 +82,9 @@ class ResourceWatcherTest extends TestCase
     /** @var FeatureFlagCheckerInterface|MockObject */
     private $featureFlagChecker;
 
+    /** @var core_kernel_persistence_ResourceInterface|MockObject */
+    private $resourceImplementation;
```

PHPMD (2.15.0)

Avoid excessively long variable names like $resourceImplementation. Keep variable name length under 20. (Naming Rules)

(LongVariable)